docs/editors/emacs.md

```diff
@@ -4,7 +4,7 @@ title: Emacs
 ---
 
 Metals works in Emacs thanks to the
-[`lsp-mode`](https://github.com/emacs-lsp/lsp-mode) package.
+[`lsp-mode`](https://github.com/emacs-lsp/lsp-mode) package (another option is the [Eglot](#eglot) package).
```

Comment:
Thought it might be a good idea to link the Eglot integration here at the top as well, since it's a fair length later down the page.